npm/private/test/defs_checked.bzl

```diff
@@ -1,4 +1,4 @@
-"@generated by @aspect_rules_js//npm/private:npm_translate_lock.bzl from //:pnpm-lock.yaml"
+"@generated by @aspect_rules_js//npm/private:npm_translate_lock.bzl from @//:pnpm-lock.yaml"
```

Comment:
this change will cause a bit of churn in all our vendored files in downstream repos, is it worth it?

Reply:
Its consistent now between the pnpm lock & yarn/npm lock cases. Just a comment anyway.